src/ui/map.js

```diff
@@ -531,6 +559,14 @@ class Map extends Camera {
      *
      * @param {LngLatBoundsLike | null | undefined} bounds The maximum bounds to set. If `null` or `undefined` is provided, the function removes the map's maximum bounds.
      * @returns {Map} `this`
+     * @example
+     * // Define bounds that conform to the `LngLatBoundsLike` object.
+     * var bounds = [
+     *   [-74.04728, 40.68392], // Southwest coordinates
```

Comment:
People often get thrown off by the coordinates order, I know we already mention this is LngLat, but for absolute clarity it might be best to change the comment here to something which is more explicit, eg.

```
// [west, south]
// [east, north]
```

Or something like that?

The numbers in there are helpful for someone working in the geographic area of the coordinates as they'll likely recognise the order from the numbers alone, but generally not helpful for an international audience, so we can't rely on those alone. eg. if I see "150, -34" I know that's `long, lat` order, but to someone in the US that's probably not helpful.